bigbluebutton-html5/imports/ui/components/breakout-room/breakout-room/running-room/component.tsx

```
@@ -0,0 +1,588 @@
+      const assignments: Record<string, { roomSequence: number; userName: string }> = JSON.parse(raw);
+      sessionStorage.removeItem('breakoutInitialAssignments');
+      initialAssignmentsLoaded.current = true;
+
+      const seqToMeetingId = new Map<number, string>();
+      breakouts.forEach((b) => {
+        seqToMeetingId.set(b.sequence, b.breakoutRoomMeetingId);
+      });
+
+      const alreadyAssigned = new Set<string>();
+      breakouts.forEach((b) => {
+        b.participants.filter((p) => !p.isAudioOnly).forEach((p) => alreadyAssigned.add(p.userId));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)